functions/routes/ecom/modules/create-transaction.js

```diff
@@ -1,3 +1,49 @@
-exports.post = ({ appSdk }, req, res) => {
-  res.send({})
+exports.post = ({ appSdk, admin }, req, res) => {
+  /**
+   * Requests coming from the modules receive a hydrated body with two objects, `params` and application`.
+   * In `application` is a copy of your application installed by the merchant, including the properties` data` and `hidden_data`.
```

Comment:
```suggestion
   * `application` is a copy of your app installed by the merchant,
   * including the properties `data` and `hidden_data` with admin settings configured values.
```

Typo fix a exemplo do `apply-discount`...